.pre-commit-config.yaml

```diff
@@ -3,11 +3,28 @@ default_language_version:
     python: python3
 
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-byte-order-marker
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: check-toml
+      - id: check-yaml
+      - id: detect-private-key
```

Comment:
This hook could have caught this: https://github.com/run-llama/llama_index/pull/7885#discussion_r1340713408